h2o-algos/src/test/java/hex/grid/GridTest.java

```diff
@@ -4,11 +4,13 @@
 import hex.genmodel.utils.DistributionFamily;
 import hex.tree.gbm.GBMModel;
 import org.junit.Before;
+import org.junit.Ignore;
```

Comment:
Appears unused. Which is a good thing :)

Reply:
@michalkurka fixed